src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -247,3 +247,6 @@ experimental=@DEFAULTEXPFEATURES@
 # If enabled, user can run pprof tools with shim v2 process through kata-monitor.
 # (default: false)
 # enable_pprof = true
+
+static_resource_mgmt=@DEFSTATICRESOURCEMGMT_DB@
```

Comment:
Please add a description for this configuration item.

Reply:
Fixed